sdk-extensions/otproto/src/main/java/io/opentelemetry/sdk/extension/otproto/TraceProtoUtils.java

```diff
@@ -21,7 +21,9 @@ private TraceProtoUtils() {}
    *
    * @param spanId the spanId to convert.
    * @return a ByteString representation.
+   * @deprecated Will be removed without replacement
    */
+  @Deprecated
```

Comment:
Is it worth also deprecating the class, as well? Not sure that it makes much difference, but just thought I'd ask.